content/docs/started/workstation/appliances/getting-started-minikube.md

```diff
@@ -206,6 +206,10 @@ $ minikube delete
 $ minikube start --cpus 4 --memory 8096 --disk-size=40g
 ```
 
+<!-- ------- ------- ------- ------- ------- ------- ------- ------- ------- ------- -->
+<!-- At this point, the user should be directed to "installing on an existing cluster" -->
+<!-- ------- ------- ------- ------- ------- ------- ------- ------- ------- ------- -->
+
```

Comment:
Is this something you're still planning to do on this page? Either remove the comment or add the required link.

Reply:
I have added the link to the "existing kubernetes cluster" getting started, and have removed the outdated instructions here.